sky/shell/android/org/domokit/sky/shell/SkyActivity.java

```diff
@@ -98,6 +98,12 @@ protected void onPause() {
     }
 
     @Override
+    protected void onResume() {
+        super.onResume();
+        ActivityImpl.updateSystemUiVisibility(this);
```

Comment:
I'd rename `updateSystemUiVisibility` to `onResumeActivity` so that `ActivityImpl` would feel free to do more work like this in the future.